src/spsc.rs

```diff
@@ -464,8 +597,8 @@ impl<T, S: Storage> QueueInner<T, S> {
     ///     consumer.dequeue().unwrap();
     /// }
     /// ```
-    pub const fn split(&mut self) -> (ProducerInner<'_, T, S>, ConsumerInner<'_, T, S>) {
-        (ProducerInner { rb: self }, ConsumerInner { rb: self })
+    pub const fn split_const(&mut self) -> (Producer<'_, T>, Consumer<'_, T>) {
```

Comment:
The example here doesn't actually call this on `QueueView`. In fact, I don't think this makes sense on `QueueView`, since we first need a static `Queue` anyways, so there is no point in going through a `QueueView` to split it.

Reply:
It makes sense to have it available for `QueueView` if you have some `const` functions that does some initialization, which you use to build the statics.

I updated the doc to use the View.

Reply:
@reitermarkus Are you satisfied with @sosthene-nitrokey's response? If so, I believe we can get a 0.9 release now?